src/main.rs

```diff
@@ -11,9 +11,110 @@ use std::marker::PhantomData;
 use french_numbers::*;
 
     /// These constants are to avoid magic strings/values.
-    const LANGUAGE_LOCALES: &[&str; 18] = &["en", "es", "bg", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ru", "sk", "tr", "zh"];
+    const LANGUAGE_LOCALES: &[&str; 19] = &["en", "es", "bg", "bw", "de", "eo", "fr", "gr", "hi", "ie", "jp", "la", "nl", "pl", "pt", "ru", "sk", "tr", "zh"];
     const LANGUAGES_DIRECTORY: &str = "translations";
     const MSG: &str = "msg";
+    
+    /// These constants are necessary numerical values. Our extensive list has 100 integers, so far.
+    const one: i64 = 1;
```

Comment:
I suggest using
```rust
const zero: i64 = std::Default::default();
```
and build everything else from this base implementation that is battle-tested and known to be blazingly performant.

Reply:
This would also solve the security problems pointed out by @elijaharita.